Makefile

```diff
@@ -13,7 +13,7 @@ DEIS_REGISTRY ?= ${DEV_REGISTRY}
 RC := manifests/deis-${SHORT_NAME}-rc.yaml
 SVC := manifests/deis-${SHORT_NAME}-service.yaml
 SEC := manifests/deis-${SHORT_NAME}-secretAdmin.yaml
-IMAGE := ${DEIS_REGISTRY}/${SHORT_NAME}:${VERSION}
+IMAGE := ${DEIS_REGISTRY}/${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
```

Comment:
There are some compelling reasons to take out the first `/` on this line and require a trailing slash in any `DEIS_REGISTRY` value instead.  The main reason is that if `DEIS_REGISTRY` is the empty string, then the image name with start with `/` and that's invalid.  This turns out to be very relevant when wishing to put to Docker hub.